index.js

```diff
@@ -1,8 +1,9 @@
-/* jshint node: true */
+/* eslint-env node */
+
 'use strict';
 
 var RSVP = require('rsvp');
-var fs = require('fs');
+var fs = require('fs-extra');
```

Comment:
yeah my mistake i thought you were talking about adding fs as a dependency still